winterfell/controllers/authController.js

```diff
@@ -92,25 +93,96 @@ module.exports = function (app) {
     });
   });
 
-  // Endpoint for routing idme authorization
+  // Server endpoint for redirecting from ID.me authorization
   // https://api.id.me/oauth/authorize?...
   app.get('/auth/link/idme', function(req, res) {
-    console.log('[authLinkIdMe] link request with ' + JSON.stringify(req.params));
-    var code = req.params.code;
+    var prodEnv = app.environment == constants.environment.PROD;
+    var baseUri = prodEnv ? 'https://www.vetafi.org' : 'http://localhost:3999';
+    var clientId = prodEnv ? '71ffbd3f04241a56e63fa6a960fbb15e' : '684c7204feed7758b25527eae2d66e28';
+    var secretClientId = prodEnv ? 'some-secret' : '57ebec28ad4bae403d0a2702f2f81801';
```

Comment:
ok cool, we can release the above 3 lines with calls to `biscuitService` after my pull is landed